docs/rules.rst

```diff
@@ -29,9 +29,11 @@ Rules
   * - Severity
     - Rule id
     - Robot Framework version
+    - Robocop version
```

Comment:
This may not be very clear for the users what is means. It may suggest that the rule is only available for this specific Robocop version.
I would rephrase it to `New from Robocop` or `Added in Robocop version` or `Introduced in Robocop (version)`. I am not 100% sure if the word `version` is needed here or not (or maybe added in parenthesis?).

Reply:
Maybe let's go with ``New from Robocop``, without version